pkg/mesh/topology.go

```diff
@@ -90,6 +91,21 @@ type segment struct {
 	allowedLocationIPs []net.IPNet
 }
 
+// CIDRS returns the cidrs of the segment.
+func (s Segment) CIDRS() []*net.IPNet {
```

Comment:
Im still a bit on the fence about making these a methods or simply exposing all of the fields of the segment. I think that exposing the fields would be easier + it doesn't make it a "weaker" contract at all. Going forward, all users of this library can expect to be able to find this data from a segment, but using the field directly seems a bit easier to me.

Reply:
fine with me to expose them. Or we move the routes function defined in kgctl's main package into the mesh package?

Reply:
Ah interesting. Like a PeerRoutes method? I like that too

Reply:
This way we can have tests